### PR TITLE
Updated process.md

### LINKED ISSUE #5603: Inaccurate explanation of closing issue with PR in process.md
This is from a section in `process.md`:

```
In the PR description, mention the issue number in this format: Fixes #1760.
Doing so will create an automatic reference from the issue to the pull request.
```

This is _inaccurate_: automatic reference is achieved by simply mentioning the issue number with the `#`. What is intended here is [closing the issue when the PR is merged](https://github.com/blog/1506-closing-issues-via-pull-requests).

### PR DESCRIPTION
**Pull request by AakanxuShah**
Fixes #5603 
 Inaccurate explanation of closing issue with PR in process.md

### CODE REVIEW

**Comment by AakanxuShah** [comment]
Source : Closes Issues via PRs - https://github.com/blog/1506-closing-issues-via-pull-requests
Please feel free to let me know if I need to change/improve anything in the article !

**Comment by chowyb** [comment]
Hi @AakanxuShah, the description for this PR is correct, however as the branch name is incorrect, and it is not possible to change a branch name in a PR, I will be closing this PR. Feel free to take a look at the [process document](https://github.com/TEAMMATES/teammates/blob/master/devdocs/process.md#fixing-issues), and refer to step 6 on the naming convention of the branch.

Also, please note that your PR title should follow the format as given in step 8 of the same document.

Good work though, and I'm looking forward to your next PR! :)

File: devdocs/process.md [hunk 1]
```diff
@@ -129,7 +129,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      the relevant issue name and include the issue number as well.<br>
      e.g. ` Incorrect error message when adding an existing instructor #1760`<br>
      In the PR description, mention the issue number in this format: `Fixes #1760`. 
-     Doing so will create an automatic reference from the issue to the pull request.<br>
+     Doing so will achieve an automatic reference (by simply mentioning the issue number with the #).<br>
+     To close an issue from a PR, all you have to do is include the special keyword syntax (eg. "fixes #5") 
+     in the body of your Pull Request. And the referenced issue will automatically be closed when the PR is merged into the default branch.<br>
```
Comment (wkurniawan07) [inline devdocs/process.md]: No, this is not what I had in mind. The change should be from the "create an automatic ... pull request" to "close the related issue once the pull request is merged".

Comment (damithc) [inline devdocs/process.md]: We can keep this description brief (as @wkurniawan07 has suggested) but it is ok to hyper link to the relevant section of GitHub's official documentation.

**Comment by AakanxuShah** [comment]
# ReadMe : 
## Here, the required task is to build a generic parallel sort and parallel join algorithm.
**1)** Implemented a Python function **ParallelSort()** that takes as input: (1) InputTable stored in
a PostgreSQL database, (2) SortingColumnName the name of the column used to order
the tuples by. 

ParallelSort() then sorts all tuples (using five parallelized threads) and
stores the sorted tuples for in a table named OutputTable (the output table name is
passed to the function). The OutputTable contains all the tuple present in InputTable
sorted in ascending order.

OutputTable – Name of the table where the output needs to be stored.
openconnection – connection to the database.

**2)** Implement a Python function **ParallelJoin()** that takes as input: (1) InputTable1 and
InputTable2 table stored in a PostgreSQL database, (2) Table1JoinColumn and
Table2JoinColumn that represent the join key in each input table respectively.

ParallelJoin() then joins both InputTable1 and InputTable2 (using five parallelized
threads) and stored the resulting joined tuples in a table named OutputTable (the output
table name is passed to the function). 

The schema of OutputTable should be similar to the schema of both InputTable1 and InputTable2 combined.

**Pull request closed without merging**